### DISCUSSION TITLE
Installation issues

### DISCUSSION THREAD

**Discussion by MaryOikonomou** [joomengine/Joomla-Component-Builder#966] [Extention Issues] [answered]
- Hello, i'm trying to install "COM_COMPONENTBUILDER 4.0.1 version" and i have these issues:

1. Joomla\CMS\Filesystem\Folder::delete: Path is not a folder. Path: [ROOT]/administrator/components/com_componentbuilder
2. Joomla\CMS\Filesystem\Folder::delete: Path is not a folder. Path: [ROOT]/api/components/com_componentbuilder
3. Joomla\CMS\Filesystem\Folder::delete: Path is not a folder. Path: [ROOT]/components/com_componentbuilder
4. Component Uninstall: Can't uninstall. Please remove manually.
Any ideas?

**Comment by Llewellynvdm** [comment, marked as answer]
Please take all issue to: https://git.vdm.dev/joomla/Component-Builder/issues/new/choose
